src/graphql/resolvers/projectResolver.ts

```diff
@@ -0,0 +1,33 @@
+import {
+	createProject,
+	getProject,
+	updateProject,
+} from "../../helpers/projectService";
+import { Arg, Int, Mutation, Query, Resolver } from "type-graphql";
+import { CreateProjectInputModel } from "../models/ProjectModel";
+import { ProjectModel, UpdateProjectInputModel } from "../models/projectModel";
+
+@Resolver()
+export class ProjectResolver {
+	@Query((_returns) => [ProjectModel])
+	async project() {
```

Comment:
plural

Reply:
should it be projects?

Reply:
yes since its returning multiple projects. 
if it was returning a single project (for example, get project by ID) then the query name must be "project".